.gitignore

```diff
@@ -7,4 +7,4 @@ war
 .gradle
 installer/appEngineSDK.zip
 gradle.properties
-logs
```

Comment:
I failed to add the pg_data line in the `.gitignore` file. Fixed in the next commit